book-src/tour/custom-types.md

```diff
@@ -250,3 +250,23 @@ LoggedIn("Kim")
 guest,
 {logged_in, <<"Kim">>}.
 ```
+
+## JavaScript interop
+
+When compiling to JavaScript, each constructor becomes a class of the same name.
+Each of the generated JavaScript classes has a constructor with the same
+arguments as the original type constructor.
+
+```gleam
+// Gleam
+Guest
+LoggedIn("Kim")
+Cat(name: "Jinx", cuteness: 2002)
+```
+
+```javascript
+// JavaScript
+new Guest();              // => {}
+new LoggedIn("Kim");      // => { 0: "Kim" }
+new Animal("Jinx", 2002); // => { name: "Jinx", cuteness: 2002 }
```

Comment:
Could you remove these comments? They're not accurate, and I think the constructors on the left are clear enough.

Reply:
Yeah, I was trying to find a way to explain how to access named and unnamed properties. Maybe we could do something like

```javascript
const user = new LoggedIn("Kim");
user[0] === "Kim";

const cat = new Cat("Jinx", 2002);
cat.cuteness === 2002;
```

Not sure if that's overkill